odl/set/pspace.py

```diff
@@ -256,15 +258,16 @@ def element(self, inp=None):
         if inp is None:
             inp = [space.element() for space in self.spaces]
 
-        # TODO: how does this differ from "if inp in self"?
-        if (all(isinstance(v, LinearSpaceVector) for v in inp) and
-                all(part.space == space
-                    for part, space in zip(inp, self.spaces))):
+        if (all(isinstance(v, LinearSpaceVector) and v.space == space
```

Comment:
`isinstance(v, LinearSpaceVector) and v.space == space`

should ofc simply be

`v in space`